docs/serving/services/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Knative Services"
+linkTitle: "Services"
+weight: 10
+type: "docs"
+---
+
+{{% readfile file="README.md" relative="true" markdown="true" %}}
```

Comment:
This link pulls in a "README.md" file from this `/services/` directory (and no longer requires 'relative' and 'markdown' flags). We do this to hide the README filenames from the URL (bad SEO).
```suggestion
{{% readfile file="README.md" %}}
```

If there is not a README file, we should remove this line. 

Alternatively, you also have the option to add an overview statement in the README.md file to define this section. (we continue to use READMEs to cater to our "GitHub users" (those  who choose to view the source files from GitHub), since the READMEs render by default when you click a directory in GitHub).